include/eve/detail/shuffle_v2/simd/arm/neon/shuffle_l3.hpp

```diff
@@ -0,0 +1,69 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
```

Comment:
vtbl is also used in lookup as it's exactly the same thing, should we sse to factor those out later ?